specification/authorization/resource-manager/Microsoft.Authorization/preview/2019-08-01-preview/authorization-UsageMetricsCalls.json

```diff
@@ -0,0 +1,105 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AuthorizationManagementClient",
+    "version": "2019-08-01-preview",
```

Comment:
>"version": "2019-08-01-preview", [](start = 4, length = 32)

I think there is have another PR that adds this version [https://github.com/Azure/azure-rest-api-specs/pull/13119](https://github.com/Azure/azure-rest-api-specs/pull/13119). Did you intend to add it in this PR too?

Reply:
Yes, they're exactly the same definition, I just included it here since this version references it and I wanted to submit both PRs